CMakeLists.txt

```diff
@@ -310,7 +310,7 @@ if(${SUBSURFACE_TARGET_EXECUTABLE} MATCHES "MobileExecutable")
 	target_link_libraries(
 		${SUBSURFACE_TARGET}
 		subsurface_profile
-		subsurface_models
+                subsurface_models_mobile
```

Comment:
that looks like whitespace problems, doesn't it?
As do many of the other changes. Can you check if you switched from tabs to spaces or vice-versa?

Reply:
yes, will check this, and correct. Probably my auto formatting stuff in QtCreator messes things up on .txt files.